src/api/BioLink.js

```diff
@@ -296,6 +296,17 @@ export async function getSources() {
   const params = new URLSearchParams();
 
   const bioentityResp = await axios.get(url, { params });
+
+  // remove this after we get this stuff from the API
+  for (let i = 0; i < bioentityResp.data.length; i++) {
+    let sourceDisplayName = bioentityResp.data[i].id.split(/[:.]+/)[1];
+    sourceDisplayName = sourceDisplayName.charAt(0).toUpperCase() + sourceDisplayName.slice(1);
+    bioentityResp.data[i].sourceDisplayName = sourceDisplayName;
+    bioentityResp.data[i].ttlUrl = 'https://data.monarchinitiative.org/ttl/' + bioentityResp.data[i].id.split(':')[1];
```

Comment:
Okay for now. For future iterations, would be good to make use of CURIE expansion instead such that MonarchData expands to a URI as defined by a JSON-LD context.

Reply:
Agreed. Is there a helper fxn to do CURIE expansion in the UI code somewhere?

Reply:
Not that I am aware of. Not a blocker for this PR. 👍